kratos/input_output/vtk_output.cpp

```diff
@@ -432,11 +428,8 @@ void VtkOutput::WriteConditionResultsToFile(const ModelPart& rModelPart, std::of
     const auto& r_local_mesh = rModelPart.GetCommunicator().LocalMesh();
     Parameters condition_results = mOutputSettings["condition_data_value_variables"];
 
-    int num_elements = static_cast<int>(r_local_mesh.NumberOfElements());
-    rModelPart.GetCommunicator().SumAll(num_elements);
-
-    int num_conditions = static_cast<int>(r_local_mesh.NumberOfConditions());
-    rModelPart.GetCommunicator().SumAll(num_conditions);
+    int num_elements = rModelPart.GetCommunicator().GetDataCommunicator().SumAll(static_cast<int>(r_local_mesh.NumberOfElements()));
```

Comment:
just wondering if it would not make sense to have a usigned int version of this

Reply:
It would, but with the current design it would require to duplicate the entire `int` interface with overloads... I'm thinking of moving to a templated interface in the future, but I still need to figure out some details.